db/migrate/20150128155644_allow_null_for_adviser_covers_whole_of_uk.rb

```diff
@@ -0,0 +1,5 @@
+class AllowNullForAdviserCoversWholeOfUk < ActiveRecord::Migration
+  def change
+    change_column :advisers, :covers_whole_of_uk, :boolean, null: true, default: nil
```

Comment:
`nil` 'cause we need the radio buttons that are tied to it to not be selected at all for new advisers. Any other way to ensure this?